src/goto-instrument/contracts/dynamic-frames/dfcc_spec_functions.cpp

```diff
@@ -272,13 +272,6 @@ void dfcc_spec_functionst::to_spec_assigns_function(
 
   goto_model.goto_functions.update();
 
-  // instrument for side-effects checking
-  std::set<irep_idt> function_pointer_contracts;
-  instrument.instrument_function(function_id, function_pointer_contracts);
-  INVARIANT(
-    function_pointer_contracts.empty(),
-    "discovered function pointer contracts unexpectedly");
-
```

Comment:
We previously had that hot-fix re-instating this piece of code. Why is it now safe to remove this?

Reply:
because the wrong removal was accidentally pulled from this commit